test/restart/conftest.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This file configures pytest for all tests in this directory, and also
+# defines common test fixtures for all of them to use
+
+import sys
+import pytest
+
+# Use the util.py library from ../cql-pytest:
+sys.path.append(sys.path[0] + '/../cql-pytest')
+from util import unique_name
```

Comment:
I'm not thrilled with the new fashion that we add a new test "suite" every almost day now.  We have one suite for 4-node (isn't it 3-node, by the way) clusters, another suite for "upgrades" and now for "restart". Why can't all these different tests be in the same suite, just require different clusters (the proper way to do this is fixtures, but @kbr-'s "manager" thing would also work) - one test asks for a 1-node cluster, another test asks for a 3-node cluster, and so on?
@kbr- @alecco what do you think?

Reply:
Two other thoughts:
1. The "topology" suite is designed to allow the size of the cluster to be controlled. So why can't a 1-node cluster be created like you want, without a new test suite? Why must the initial size of every cluster be a "yaml" parameter instead of something settable in Python code (e.g., a fixture)?
2. If, despite what I wrote above, the difference between the "topology" and "restart" suites should remain and is just the number of nodes in the cluster - then the names "topology" and "restart" are the wrong names... What if we later want other single-node tests that don't involve restart but some other cluster operation (e.g., perhaps adding another node, or I don't know what)?

Reply:
I discussed already with @kostja about having dynamic cluster size. I don't mind either way. Neither of these were my ideas.

Reply:
If I understand correctly, you already have a dynamic cluster size - it's just that when the test asks for a "manager" it gets one with a fixed number of nodes specified in the yaml. I don't see why it needs to be like that. Why can't the test specify explicitly how many nodes it needs? In fact, I think it should always specify the number of nodes - I don't understand why a default number of nodes is useful.

Reply:
I think it got to be a separate patch. The clusters are pooled, and allowing arbitrary size may break (or make difficult) cluster recycling.

Reply:
Considering the specificity of the `topology` tests, I won't add this `restart` test there. Creating an entire new directory for this single test also doesn't sound good, so I'll just add it to dtests